rollup.config.js

```diff
@@ -0,0 +1,33 @@
+import commonjs from 'rollup-plugin-commonjs';
+import resolve from 'rollup-plugin-node-resolve';
+import modify from 'rollup-plugin-modify';
+
+export default [
+  {
+    input: 'index.js',
+    output: {
+      file: 'dist/focus-trap.mjs',
+      format: 'esm'
+    },
+    plugins: [
+      modify({
+        // swap tababble require load to ES6 import
+        "var tabbable = require('tabbable');":
+          "import tabbable from 'tabbable';",
+        // ES6 module support implicitly means Object.assign support, so xtend can drop out
+        "var xtend = require('xtend');": '',
+        'xtend(': 'Object.assign(',
+        // swap focus-trap from node module.exports to ES6 named export
+        'function focusTrap(': 'export function focusTrap(',
```

Comment:
Can browserify/rollup not handle multiple targets? Having multiple build tools seems less than ideal? I know webpack could handle generating multiple outputs of various types if wanted.

To me, the ideal pattern would be:
1) Update the src to be es6
2) Build then just minifies source to a `dist/**.mjs` file for es6 support
3) Build also runs rollup or webpack or browserify to generate the legacy commonjs support file

For the `xtend` use case, I feel like that can just be left out and library users can specify (via polyfill) how to handle `Object.assign` uses.